components/myft-concept-button/.gitignore

```
@@ -0,0 +1,9 @@
+.DS_Store
+.env
+.idea/
+/.sass-cache/
+/bower_components/
+/build/
+/demos/local
+/node_modules/
+npm-debug.log
```

```suggestion
.env
.gitignore
.npmignore
.eslintrc.cjs
.remarkrc.cjs
.stylelintrc.cjs
.prettierrc.toml
.eslintignore
.stylelintignore
stories/
src/tsx/
.github/
test/
demos/local/

```

☝️ Brings inline with other components. We need to update this in the create new component workflow at some point. https://github.com/Financial-Times/origami/pull/596

Also means the tsx/stories won't be published. But maybe we want to make this the first component with published tsx?

---

yeah, we will want to publish the tsx on this one for sure

---

👍 I've updated the suggestion to remove tsx. It still adds stories to the exclude and a couple of missing dot files

---

wait. why do we want to ignore these in the gitignore??

```
.gitignore
.npmignore
.eslintrc.cjs
.remarkrc.cjs
.stylelintrc.cjs
.prettierrc.toml
.eslintignore
.stylelintignore
```

we use those files

---

Oh no 🙈 this should go in `.npmignore`